packages/datadog-plugin-mongodb-core/src/index.js

```diff
@@ -27,8 +27,10 @@ class MongodbCorePlugin extends DatabasePlugin {
 
 function getQuery (cmd) {
   if (!cmd || typeof cmd !== 'object' || Array.isArray(cmd)) return
-  if (cmd.query) return JSON.stringify(limitDepth(cmd.query))
-  if (cmd.filter) return JSON.stringify(limitDepth(cmd.filter))
+  if (cmd.query) return JSON.stringify(limitDepth(cmd.query),2 , null)
```

Comment:
These `, 2, null` args look like a typo? Maybe made while debugging? Using `, null, 2` aids in readability by adding newlines and indentation, so I think the arg order was switched. Also I don't think we'd want to commit that change as the strings would be larger.